client/state/themes/hooks/use-is-theme-allowed-on-site.ts

```diff
@@ -42,3 +35,12 @@ export function useIsThemeAllowedOnSite( siteId: number | null, themeId: string
 	return hasFeature;
 	 */
 }
+
+export function isThemeAllowedOnSite( siteId: number | null, themeId: string, state: IAppState ) {
```

Comment:
This should be moved from the `hooks` dir to the `selectors` dir: https://github.com/Automattic/wp-calypso/tree/trunk/client/state/themes/selectors

Also, the first param in selectors is typically the state:
```suggestion
export function isThemeAllowedOnSite( state: IAppState, siteId: number | null, themeId: string ) {
```

Reply:
Updated